pyat/at/lattice/utils.py

```diff
@@ -832,7 +832,8 @@ def get_value_refpts(
         attrname:   Attribute name
         index:      index of the value to retrieve if *attrname* is
           an array. If :py:obj:`None` the full array is retrieved
-          In case *attrname* is a scalar index has to be None
+          In case *attrname* is a scalar, the keyword argument index has
+          to be None
```

Comment:
I repeat my proposal:
```
In case *attrname* is a scalar attribute, index has to be None
```
Shorter and clearer

Reply:
I kept keyword argument index to clear any ambiguity